### ISSUE TITLE
Implement proper Webhook models

### ISSUE THREAD

**Issue by gambry** [iboldurev/api-ai-php#10]
Hi,

I've been browsing this library for the last couple of days trying to understand how to implement webhooks handling, without much luck.

I made a [PR](https://github.com/iboldurev/api-ai-php/pull/9) in order to add Webhook Request and Response models.

**Comment by gambry** [comment]
https://github.com/iboldurev/dialogflow/pull/9 has been merged, this issue can now be closed.

**Comment by nicola-rossato-opensymbol** [comment]
Hi @gambry, I need to know if is possible to query DialogFlow specifying the action to use, have you an example of this? Thank you

**Comment by gambry** [comment]
No, I haven't worked with action yet. I can see there is a model for that in this library, but haven't tried it yet. Sorry.

**Comment by nicola-rossato-opensymbol** [comment]
No matter, thank you anyway for your reply :]
